main.go

```
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/json"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"github.com/ufcg-lsd/arrebol-pb-worker/worker"
+	"log"
+	"net/http"
+	"os"
+)
+
+func setup(serverEndPoint string, workerId string) {
+	// The setup routine is responsible for generates the rsa key pairs
+	// of the worker and to send the public part to the server.
+	log.Println("Starting to gen rsa key pair with workerid: " + workerId)
+
+	utils.Gen(workerId)
+
+	log.Println("Sending pub key to the server")
+	url := serverEndPoint + "/workers/publicKey"
+	requestBody, err := json.Marshal(&map[string]*rsa.PublicKey{"key": utils.GetPublicKey(workerId)})
```

Any reason to ignore the error?